docs/theme/templates/layout.mustache

```diff
@@ -250,6 +250,18 @@
 	 ga('create', 'UA-2302003-14', 'auto');
 	 ga('send', 'pageview');
 	</script>
+	
+	<!-- Start of Async HubSpot Analytics Code -->
+  <script type="text/javascript">
+    (function(d,s,i,r) {
+      if (d.getElementById(i)){return;}
+      var n=d.createElement(s),e=d.getElementsByTagName(s)[0];
+      n.id=i;n.src='//js.hs-analytics.net/analytics/'+(Math.ceil(new Date()/r)*r)+'/2171535.js';
```

Comment:
It looks like this is trying to prevent the script from being cached, I wonder why that is?